README.md

```diff
@@ -31,31 +31,30 @@ Use "astro [command] --help" for more information about a command.
 
 ## Installing `astro`
 
-### If you're using Astronomer Cloud or Astronomer Enterprise v0.7.x
+The Astronomer CLI is an open-source project. Installing it to your machine allows you to easily spin up a local instance of Apache Airflow and allows you to easily deploy code to remote Airflow environments if you are an Astronomer customer.
 
-via `curl`:
+> Note: If you are an Astronomer customer, your CLI version _must_ match the version of Astronomer you are running. If you are using Astronomer Cloud, the latest version should always be correct. If you have a custom Astronomer Enterprise installation, you may be on a different version, which means you may need to ensure that your CLI and platform match up; you can check which version of Astronomer you're running by clicking the user icon in the top right corner of our UI.
 
-```
-curl -sSL https://install.astronomer.io | sudo bash -s -- v0.7.5
-```
+### Latest Version
 
-### If you're using Astronomer Enterprise v0.8 or later
+via `curl`:
 
-via `curl`
 ```
-curl -sSL https://install.astronomer.io | sudo bash
+curl -sSL https://install.astronomer.io | sudo bash -s
 ```
 
 ### Previous Versions
 
-via `curl`
+
+To install a previous version of Astronomer, you can add the tag at the end of your `curl` command via the following syntax:
+
 ```
 curl -sSL https://install.astronomer.io | sudo bash -s -- [TAGNAME]
 ```
 
 ie:
 ```
-curl -sSL https://install.astronomer.io | sudo bash -s -- v0.3.1
+curl -sSL https://install.astronomer.io | sudo bash -s -- v0.7.5
```

Comment:
what do you think about 0.10.3?

Reply:
Actually lets just drop the version here so it installs the latest stable by default. And we should change the url to be the new prod url for this.

Reply:
So this is a section specifically for "Previous Versions". It comes after the "Latest Version" section which installs the latest stable by default.

What's the new prod URL?

Reply:
That's still it. There's always one launched with the platform that installs the platform version, but this on is separate and hosted on its own. It'll always do the latest published.